signing_service/signing_service/tests/tests.py

```diff
@@ -116,6 +131,267 @@ def test_that_signing_service_should_reraise_different_socket_erros(self):
         mock_socket_close.assert_called_once()
 
 
+class SigningServiceHandleConnectionTestCase(SigningServiceIntegrationTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.host = '127.0.0.1'
+        self.port = 8000
```

Comment:
port should not be fixed - please use `unused_tcp_port` fixture

Reply:
Done in base branch https://github.com/golemfactory/concent/pull/688.